javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/PropagatorsInitializer.java

```diff
@@ -50,15 +70,31 @@ public static void initializePropagators(List<String> propagators) {
       OpenTelemetry.setGlobalPropagators(
           DefaultContextPropagators.builder()
               .addTextMapPropagator(HttpTraceContext.getInstance())
+              .addTextMapPropagator(W3CBaggagePropagator.getInstance())
               .build());
       return;
     }
 
     DefaultContextPropagators.Builder propagatorsBuilder = DefaultContextPropagators.builder();
 
     List<TextMapPropagator> textPropagators = new ArrayList<>(propagators.size());
-    for (String propagatorId : propagators) {
-      TextMapPropagator textPropagator = TEXTMAP_PROPAGATORS.get(propagatorId.trim().toLowerCase());
+    List<String> propagatorIds =
+        propagators.stream()
+            .map(propagator -> propagator.trim().toLowerCase())
```

Comment:
I don't see mention of supporting case-insensitivity in [the spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/sdk-environment-variables.md), I would suggest leaving this out?

Reply:
it was from beginning I suppose, didn't want change behavior

Reply:
ok, I created #1632 to track this